File: TapyrusWalletSwift/example/example/ConnectionSettings.swift
//
//  ConnectionSettings.swift
//  example
//

import Foundation

enum ConnectionType: String, CaseIterable {
    case esplora
    case electrum
}

enum SettingsKey {
    static let connectionType = "connectionType"
    static let esploraHost = "esploraHost"
    static let esploraPort = "esploraPort"
    static let electrumHost = "electrumHost"
    static let electrumPort = "electrumPort"
}

enum DefaultConnection {
    static let esploraHost = "localhost"
    static let esploraPort = "3001"
    static let electrumHost = "localhost"
    static let electrumPort = "50001"
}

File: TapyrusWalletAndroid/example/app/src/main/java/com/chaintope/tapyrus/wallet/example/ConnectionSettings.kt
package com.chaintope.tapyrus.wallet.example

enum class ConnectionType(val label: String) {
    ESPLORA("Esplora"),
    ELECTRUM("Electrum");
}

object SettingsKey {
    const val CONNECTION_TYPE = "connectionType"
    const val ESPLORA_HOST = "esploraHost"
    const val ESPLORA_PORT = "esploraPort"
    const val ELECTRUM_HOST = "electrumHost"
    const val ELECTRUM_PORT = "electrumPort"
}

object DefaultConnection {
    const val ESPLORA_HOST = "10.0.2.2"
    const val ESPLORA_PORT = "3001"
    const val ELECTRUM_HOST = "10.0.2.2"
    const val ELECTRUM_PORT = "50001"
}
